app/helpers/concerns/foreman_bootdisk/hosts_helper_ext.rb

```diff
@@ -19,6 +20,15 @@ def host_title_actions_with_bootdisk(*args)
     host_title_actions_without_bootdisk(*args)
   end
 
+  # need to wrap this one in a test for template proxy presence
+  def display_bootdisk_for_subnet
+    if (proxy = @host.try(:subnet).try(:tftp)) && proxy.features.map(&:name).include?('Templates')
```

Comment:
can we extract it to a method? it repeats a few times and we might not want to relay on tftp feature..